web/docs/about-vast/vision.md

```diff
@@ -0,0 +1,157 @@
+security operations architecture. Our goal is to break up security silos by
+decomposing monolith systems into modular building blocks that can be flexibly
+recomposed with a best-of-breed mindset. We commit to open, standardized
+interfaces at the core that prevent vendor lock-in, both for security content
+(e.g., [STIX][STIX], [CACAO][CACAO]) and structured event data (e.g., [Apache
+Arrow][Arrow]).
+
+[STIX]: https://oasis-open.github.io/cti-documentation/stix/intro.html
+[CACAO]: http://docs.oasis-open.org/cacao/security-playbooks/v1.1/security-playbooks-v1.1.html
+[Arrow]: https://arrow.apache.org
+
+Our conviction is that this requires a fundamentally different approach to
+security operations center (SOC) architecture. Conceptually, we need to shift
+away from point-to-point product integrations built on top of narrow custom APIs
+to an open *security data fabric* as central driver of use cases. This fabric
+abstracts away the complexity of the infrastructure and provides connectivity
+from cloud to distal on-premise locations, but it also features modular
+functions that can easily be composed into use cases in the detection and
+response realm. Authorized parties join this fabric to announce their
+capabilities. Based on a common ontological understanding, operators merely have
+to connect applications to the fabric to yield an autonomously acting system.
+
+The diagram below illustrates the core idea of this architectural shift that
+moves away from point-to-point integrations:
+
+![Security Data Fabric](/img/security-data-fabric.light.png#gh-light-mode-only)
+![Security Data Fabric](/img/security-data-fabric.dark.png#gh-dark-mode-only)
+
+The communication principle of the fabric is publish-subscribe, wrapping
+request-response-style communication where appropriate. An example scenario
+looks as follows: a network sensor publishes structured events to the fabric. A
+detector subscribes to this data stream to publish alerts back to the fabric. A
+triaging engine consumes to the alerts, inventory, and vulnerability information
+to create prioritized incidents in a case management tool.
+
+:::note Beyond Message Bus
+We are arguing for more than just a generic message bus here. Large
+organizations have enough engineering bandwidth to [build their own custom
+version][intel-soc]. In addition to commoditizing this architecture, our goal is
+to implant security domain knowledge and analytical data processing power into
+the very fabric, such that security application developers do not have
+to deal with data wrangling and can purely focus on providing value with domain
+logic.
+:::
+
+[intel-soc]: https://www.intel.com.au/content/www/au/en/it-management/intel-it-best-practices/modern-scalable-cyber-intelligence-platform-kafka.html
+
+How do we get there? To answer this question, we first have to understand where
+we are today.
+
+## The SOC Architecture Maze
+
+Today's SOC architecture is product-centric: SIEM harbors all data, SOAR
```

Comment:
Needs disclaimer on the abbreviations and target audience for this section.

Reply:
I created a follow-up story for this.